migrations/2018-05-14-200933_setup/up.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE album (
+  id VARCHAR(36) PRIMARY KEY NOT NULL,
```

Comment:
These IDs should be `INTEGER`'s. The reason they were strings with Redis is because Redis has database-global IDs, and they're already strings. Then they can also be `AUTO INCREMENT`

Reply:
No, they're 36 character strings because they will be generated UUID's. Instead of `AUTO INCREMENT INTEGER` where you have to get the value back from the insert result. With UUIDs you just generate and insert a UUID along with the rest of the data.

Maybe the UUIDs should be encoded as an integer though but that adds an encoding and decoding step. Let's leave it like this and if there's performance issues, we'll fix it later.

Reply:
Ok, then the 36 character length limit will need to be documented in some way so we don't get any issues with it.

Reply:
It is documented here.

Reply:
Eventually when the uuid library is imported, we'll use the uuid id type and that will enforce this constraint.